spec/system/sign_up_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+#
+# PLEASE DO NOT WRITE SLOW SYSTEM SPEC TO TEST EDGE CASES
+# MODEL AND REQUESTS SPEC ARE MUCH FASTER
+#
+RSpec.describe "Home management", type: :system do
+  before do
+    driven_by(:selenium_chrome_headless) if ENV["CI"].present?
+  end
+  it "home" do
+    visit "/"
+
+    expect(page).to have_text("Login")
+  end
+end
```

Comment:
Still browser system test, and I suggest if we are doing disabling the browser.
Can you do a request test instead?

2023-04-10 15:04:52 WARN Selenium [DEPRECATION] [:capabilities] The :capabilities parameter for Selenium::WebDriver::Chrome::Driver is deprecated. Use :options argument with an instance of Selenium::WebDriver::Chrome::Driver instead.
.

**Warning of deprecation.
Can be overridden in support folder, writing index file in rspec helper file and changin capybara settings**.

Reply:
> Can you do a request test instead? 

I agree to use requests spec for all our ruby code but currently I have not written any controller or similar ruby code.

This was just a placeholder for feature test so we do not use them to much, I suggest only a few feature tests (like sign up)

> Can be overridden in support folder, writing index file in rspec helper file and changin capybara settings.

Please write an issue and PR that solves this warning.